obspy/taup/tau.py

```diff
@@ -4,6 +4,12 @@
                         unicode_literals)
 from future.builtins import *  # NOQA
 
+import copy
+
+import matplotlib.cbook
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
For scientific packages, it's SciPy conventions, which unfortunately are not alphabetical.